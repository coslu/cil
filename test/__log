make[1]: *** [combine8] Interrupt
