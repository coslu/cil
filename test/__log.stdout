make[1]: Entering directory '/home/baro/Documents/Goblint/cil/test'
cd ./small1; \
          /home/baro/Documents/Goblint/cil/test/../bin/cilly --mode=GNUCC --decil --save-temps  --strictcheck --stats -D_GNUCC -Wall -g -ggdb -D_DEBUG -std=gnu90 \
                    combine8_2.c combine8_1.c \
            -ocombine8.exe
Makefile:279: recipe for target 'combine8' failed
